spec/jobs/person_import_job_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.describe PersonImportJob, type: :job do
+
+  before(:each) do
+    allow(PermittedDomain).to receive(:pluck).with(:domain).and_return(['valid.gov.uk'])
+  end
+
+  let(:group) { create(:group) }
+
+  let(:serialized_people) do
+    <<-CSV.strip_heredoc
+      email,given_name,surname
+      peter.bly@valid.gov.uk,Peter,Bly
+      jon.o.carey@valid.gov.uk,Jon,O'Carey
+      "Jerde, Lelah (Lelah.Jerde@valid.gov.uk)",Lelah,Jerde
+    CSV
+  end
+
+  let(:serialized_group_ids) { YAML.dump([group.id]) }
+
+  subject(:perform_later) { described_class.perform_later(serialized_people, serialized_group_ids) }
+  subject(:perform_now)   { described_class.perform_now(serialized_people, serialized_group_ids) }
+
+  it "enqueues with appropriate config settings" do
+    expect(described_class.new.queue_name).to eq 'person_import'
```

Comment:
You could do a 
`subject(:job) { described_class.new }`
to save creating 4 objects?